src/psyclone/gocean1p0.py

```diff
@@ -488,8 +471,10 @@ def create(parent, loop_type, field_name="", field_space="",
         :param str field_name: name of the field this loop iterates on.
         :param str field_space: space of the field this loop iterates on.
         :param str iteration_space: iteration space of the loop.
+        :param str index_offset: the grid index offset that this loops \
```

Comment:
Really, it is the offset used by the kernel within the loop so maybe: "...offset used by the kernel(s) within this loop."

Reply:
Done